tests/acceptance/test_organization_switch.py

```diff
@@ -0,0 +1,76 @@
+from __future__ import absolute_import
+
+from sentry.testutils import AcceptanceTestCase, SnubaTestCase
+
+
+class OrganizationSwitchTest(AcceptanceTestCase, SnubaTestCase):
+    def setUp(self):
+        super(OrganizationSwitchTest, self).setUp()
+
+        self.primary_projects = [
+            self.create_project(organization=self.organization, teams=[self.team], name=name)
+            for name in ["Bengal", "Sumatra", "Siberian"]
```

Comment:
No need to change this, but jfyi you don't have to provide a name when creating projects, they'll be autogenerated.

Reply:
Noted on this, thanks!